<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>580</width>
    <height>639</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>580</width>
    <height>340</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>580</width>
    <height>660</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Микросекундный измеритель</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QComboBox" name="osc">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>381</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="standard">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>381</width>
      <height>26</height>
     </rect>
    </property>
    <property name="statusTip">
     <string/>
    </property>
    <property name="whatsThis">
     <string/>
    </property>
    <property name="accessibleName">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="showAuxPanel">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>190</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>0</y>
      <width>191</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Адрес осциллографа</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Адрес калибратора</string>
    </property>
   </widget>
   <widget class="QPushButton" name="start">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>20</y>
      <width>151</width>
      <height>131</height>
     </rect>
    </property>
    <property name="text">
     <string>Начать</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="base">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>120</y>
      <width>101</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="top">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>120</y>
      <width>101</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="amp">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>170</y>
      <width>101</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>100</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Вершина</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>100</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Основание</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>150</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Амплитуда</string>
    </property>
   </widget>
   <widget class="QPushButton" name="record">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>190</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Отчет</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>161</width>
      <height>71</height>
     </rect>
    </property>
    <property name="title">
     <string>Импеданс источника</string>
    </property>
    <widget class="QRadioButton" name="rb1MOhm">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>74</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>1 МОм</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="rb50Ohm">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>43</y>
       <width>74</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>50 Ом</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>580</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QDockWidget" name="dockWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>51</width>
     <height>320</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>524287</width>
     <height>320</height>
    </size>
   </property>
   <property name="features">
    <set>QDockWidget::NoDockWidgetFeatures</set>
   </property>
   <attribute name="dockWidgetArea">
    <number>8</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <widget class="QCheckBox" name="averageing">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>140</y>
       <width>181</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Включить усреднение</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QSpinBox" name="averagingNum">
     <property name="geometry">
      <rect>
       <x>530</x>
       <y>140</y>
       <width>42</width>
       <height>26</height>
      </rect>
     </property>
     <property name="value">
      <number>64</number>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="interval">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>60</y>
       <width>141</width>
       <height>26</height>
      </rect>
     </property>
     <property name="suffix">
      <string> мкс</string>
     </property>
     <property name="decimals">
      <number>3</number>
     </property>
     <property name="minimum">
      <double>0.000000000000000</double>
     </property>
     <property name="maximum">
      <double>1000000.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.001000000000000</double>
     </property>
     <property name="stepType">
      <enum>QAbstractSpinBox::DefaultStepType</enum>
     </property>
     <property name="value">
      <double>0.000000000000000</double>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>50</y>
       <width>81</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Интервал</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="interChannelDev">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>270</y>
       <width>301</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Измерить межканальную погрешность</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>321</width>
       <height>171</height>
      </rect>
     </property>
     <property name="title">
      <string>Разрядность АЦП осциллографа</string>
     </property>
     <widget class="QRadioButton" name="_10bit">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>110</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>10 бит </string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QRadioButton" name="_11bit">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>110</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>11 бит</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="_12bit">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>110</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>12 бит</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="_13bit">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>110</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>13 бит</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="_14bit">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>100</y>
        <width>171</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>14 бит (300 МГц)</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="_15bit">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>120</y>
        <width>161</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>15 бит (165 МГц)</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="_16bit">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>140</y>
        <width>151</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>16 бит (82,5 МГц)</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>321</width>
       <height>91</height>
      </rect>
     </property>
     <property name="title">
      <string>Тип измерений</string>
     </property>
     <widget class="QRadioButton" name="rbAverage">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>300</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Среднее всех точек</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QRadioButton" name="rbIntervalAverage">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>42</y>
        <width>300</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Среднее на интервале</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="rbIntervalInstant">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>64</y>
        <width>300</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Мгновенные значения через интервал</string>
      </property>
     </widget>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>180</y>
       <width>81</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Сдвиг</string>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="skew">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>190</y>
       <width>141</width>
       <height>26</height>
      </rect>
     </property>
     <property name="suffix">
      <string> мкс</string>
     </property>
     <property name="decimals">
      <number>3</number>
     </property>
     <property name="minimum">
      <double>0.000000000000000</double>
     </property>
     <property name="maximum">
      <double>1000000.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.001000000000000</double>
     </property>
     <property name="stepType">
      <enum>QAbstractSpinBox::DefaultStepType</enum>
     </property>
     <property name="value">
      <double>0.000000000000000</double>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
